lang/LangPrimSource/PyrSymbolPrim.cpp

```diff
@@ -179,6 +179,82 @@ int prSymbolIsMetaClassName(struct VMGlobals *g, int numArgsPushed)
 	return errNone;
 }
 
+const char *binary_op_characters = "!@%&*-+=|<>?/";
```

Comment:
Sorry, should have been clearer—you can put them inside the function `prSymbol_IsBinaryOp` with `static` scope, and that means they are treated as init-once storage within the function but with local scope.